data/.gitignore

```diff
@@ -0,0 +1 @@
+*tsv
```

Comment:
Should probably add `*. tsv` for clarity (we don't want to ignore `xxxtsv`). Also checking that you don't want to just ignore the entire `data` directory, which you could specify in the root `.gitignore`?